src/io/flutter/view/FlutterViewFactory.java

```diff
@@ -33,9 +33,5 @@ private static void initFlutterView(@NotNull Project project, FlutterViewMessage
 
   @Override
   public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
-    //noinspection CodeBlock2Expr
-    DumbService.getInstance(project).runWhenSmart(() -> {
-      (ServiceManager.getService(project, FlutterView.class)).initToolWindow(toolWindow);
-    });
```

Comment:
I'm missing some context but I'm surprised that this is no longer needed?

Reply:
I'm missing the context on this method so I don't quite understand what is going on here.

We don't have an initToolWindow method anymore as the tool window is blank until an app starts.
I can't figure out how to locally trigger the plugin without this tool window enabled when an app starts. Maybe I'm doing something wrong. If there is a way to reach that case, then we should be initializing the tool window with all currently running applications. I'm not clear how you would get that list of running applications though.

Reply:
I added this back so I could call
toolWindow.setToHideOnEmptyContent(true)
to the initToolWindow code.
So the flutter inspector isn't shown when there is no active debug session.